src/ImageSharp/PixelFormats/PixelOperations{TPixel}.cs

```diff
@@ -85,28 +88,29 @@ internal virtual void PackFromXyzBytes(BufferSpan<byte> sourceBytes, BufferSpan<
         /// <summary>
         /// Bulk version of <see cref="IPixel.ToXyzBytes(byte[], int)"/>.
         /// </summary>
-        /// <param name="sourceColors">The <see cref="BufferSpan{T}"/> to the source colors.</param>
-        /// <param name="destBytes">The <see cref="BufferSpan{T}"/> to the destination bytes.</param>
+        /// <param name="sourceColors">The <see cref="Span{T}"/> to the source colors.</param>
+        /// <param name="destBytes">The <see cref="Buffer{T}"/> to the destination bytes.</param>
+        /// <param name="startIndex">The starting index of the <paramref name="destBytes"/>.</param>
         /// <param name="count">The number of pixels to convert.</param>
-        internal virtual void ToXyzBytes(BufferSpan<TPixel> sourceColors, BufferSpan<byte> destBytes, int count)
+        internal virtual void ToXyzBytes(Span<TPixel> sourceColors, Buffer<byte> destBytes, int startIndex, int count)
```

Comment:
I will do it soon then. Is it possible to temporally change `IPixel` API to consume `Span<byte>`+ index and rollback changes on `PixelOperations<T>` to use spans again? 
Seems easier to make the `PixelOperations<T>` rollback it in this PR, than doing it later.